cpmpy/tools/xcsp3/xcsp3_globals.py

```diff
@@ -0,0 +1,1041 @@
+"""
+Additional global constraints which are not (yet) part of the standard CPMpy collection.
+
+This file contains all the missing global constraints in order to support XCSP3-core, 
+which is a restricted scope of the complete XCSP3 specification (as used for the competitions).
+
+Currently, version 3.2 is supported.
+
+
+===============
+List of classes
+===============
+
+.. autosummary::
+    :nosignatures:
+
+    AllDifferentLists
+    AllDifferentListsExceptN
+    SubCircuit
+    SubCircuitWithStart
+    SafeOnlyInverse
+    InverseOne
+    Channel
+    NonReifiedTable
+    ShortTable
+    NegativeShortTable
+    MDD
+    Regular
+    NotInDomain
+    NoOverlap2d
+    IfThenElseNum
+    Element
+    Cumulative
```

Comment:
I think its called DynamicCumulative now; and I don't think I saw Element.

Plus the solver specific ones to add?